main.tf

```diff
@@ -108,7 +108,7 @@ module "gke" {
   network_policy                = var.gke_network_policy
   remove_default_node_pool	    = true
 
-  grant_registry_access         = var.create_container_registry
+  grant_registry_access         = var.enable_registry_access
```

Comment:
So the variable flag `create_container_registry` has been changed to `enable_registry_access` which is a bit different than we do with Azure. Is the container registry there by default in GCP and we just need to actually enable access to that CR? Asking as we've now moved away from the standard naming of the variable.

Reply:
Short answer: yet, the container registry is there by default in gcp, at https://gcr.io/<project-name>
Details:
https://cloud.google.com/container-registry/docs/overview